node/src/accountant/db_access_objects/sent_payable_dao.rs

```diff
@@ -60,6 +61,7 @@ pub trait SentPayableDao {
     fn insert_new_records(&self, txs: &[Tx]) -> Result<(), SentPayableDaoError>;
     fn retrieve_txs(&self, condition: Option<RetrieveCondition>) -> Vec<Tx>;
     fn update_tx_blocks(&self, hash_map: &TxUpdates) -> Result<(), SentPayableDaoError>;
```

Comment:
Honestly, I don't like confusion this data alias can cause, because it hides the outer structure, the sort of the collection. I'd find this information handy, so I'm thinking it makes sense to type-alias values or even keys in a hash map but if you wrap up even the HashMap type with it, it's not really helping, rather the opposite.